### PR TITLE
Bigtable: Combine create table and create column families

### PR DESCRIPTION
**Pull request by kevinsi4508**
When creating a table, Terraform first invokes Bigtable table creation and then follow and then adds column families. If a table is replicated, adding column families can be expansive and can race with table creation.

This PR combines create table and create column families into one API call. This will mitigate the race condition and timeout mentioned above.

All existing tests should cover the test cases and all of them should pass.

fixes https://github.com/hashicorp/terraform-provider-google/issues/12442

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
bigtable: combined create table and create column families into a single API call.
```

### CODE REVIEW

**Comment by kevinsi4508** [comment]
@telpirion PTAL

**Review by hoangpham95 — APPROVED** [review]

File: mmv1/third_party/terraform/resources/resource_bigtable_table.go [hunk 1]
```diff
@@ -105,42 +106,39 @@ func resourceBigtableTableCreate(d *schema.ResourceData, meta interface{}) error
...
+				columnFamilies[v.(string)] = bigtable.NoGcPolicy()
 			}
 		}
 	}
+	tblConf.Families = columnFamilies
+
+	// This method may return before the table's creation is complete - we may need to wait until
+	// it exists in the future.
+	// Set a longer timeout as creating table and adding column families can be pretty slow.
+	ctxWithTimeout, cancel := context.WithTimeout(ctx, 20*time.Minute)
```
Comment (telpirion) [inline mmv1/third_party/terraform/resources/resource_bigtable_table.go:136]: NIT: You can keep this as just `ctx` I believe.

**Review by telpirion — APPROVED** [review]

File: mmv1/third_party/terraform/resources/resource_bigtable_table.go (hunk 1 above)
Comment (kevinsi4508) [reply mmv1/third_party/terraform/resources/resource_bigtable_table.go:136]: ack

**Comment by c2thorn** [comment]
Bigtable tests are skipped in VCR tests so I'll do some due diligence and run all relevant tests manually in our CI.

**Review by c2thorn — APPROVED** [review]
Tests pass, and logic seems good although I mostly defer to the other Googlers reviewing for that. This also is limited to table creation so it should not affect existing tables. LGTM.

**Pull request merged**